src/test/ui/regions/regions-name-duplicated.stderr

```diff
@@ -2,10 +2,19 @@ error[E0263]: lifetime name `'a` declared twice in the same scope
   --> $DIR/regions-name-duplicated.rs:1:16
    |
 LL | struct Foo<'a, 'a> {
-   |            --  ^^ declared twice
+   |            --  ^^ lifetime `'a` already in scope
    |            |
-   |            previous declaration here
+   |            first declared here
 
-error: aborting due to previous error
+error[E0392]: parameter `'a` is never used
+  --> $DIR/regions-name-duplicated.rs:1:12
+   |
+LL | struct Foo<'a, 'a> {
+   |            ^^ unused parameter
+   |
+   = help: consider removing `'a`, referring to it in a field, or using a marker such as `PhantomData`
```

Comment:
I'd take this to be a regression... Could we expand E0392 to check for the duplicated case so that we don't emit this redundant error?

Reply:
The change needs to be in a totally different place. I'll do that in a follow-up PR.